README.md

```diff
@@ -16,7 +16,7 @@
 
 # style-loader
 
-Adds CSS to the DOM by injecting a <code>&lt;style&gt;</code> tag
+Adds CSS to the DOM.
```

Comment:
After rewriting tests i find style-loader inject styles not only using `style` tag :smile: